packages/gitbook/src/lib/seo.ts

```diff
@@ -55,16 +49,12 @@ export function isSpaceIndexable({
         return false;
     }
 
-    if (parent && parent.object === 'site') {
-        return shouldIndexVisibility(parent.visibility);
+    if (site && site.object === 'site') {
+        return shouldIndexVisibility(site.visibility);
     }
 
-    if (space.visibility === ContentVisibility.InCollection) {
-        return parent && parent.object === 'collection'
-            ? shouldIndexVisibility(parent.visibility)
-            : false;
-    }
-    return shouldIndexVisibility(space.visibility);
+    // space with no site should not be indexed
```

Comment:
What is a space with no site?

Or is it stuff that you want to change in another PR for the pdf export?

Reply:
yes